index.js

```diff
@@ -293,7 +293,8 @@ async function cleanup(url, options) {
 			textContent,
 			length: parsed.length,
 			siteName: sanitizer.sanitize(parsed.siteName),
-			remoteResources
+			remoteResources,
+			html: content
```

Comment:
I would call this something that has less of change of inadvertently being used in the custom HTML template, maybe `originalContent` or `unprocessedContent`.